nekoyume/Assets/_Scripts/State/LocalLayerModifier.cs

```diff
@@ -57,21 +58,19 @@ public static void ModifyAgentGold(Address agentAddress, BigInteger gold)
                 0));
         }
 
-        public static void ModifyAgentCrystal(Address agentAddress, BigInteger crystal)
+        public static async void ModifyAgentCrystalAsync(Address agentAddress, BigInteger crystal)
```

Comment:
```suggestion
        public static async UniTaskVoid ModifyAgentCrystalAsync(Address agentAddress, BigInteger crystal)
```

How about this function return `UniTaskVoid` instead of `void`? And use like below.
```
ModifyAgentCrystalAsync().Forget();
```

I recommend [this](https://github.com/Cysharp/UniTask#async-void-vs-async-unitaskvoid).

Reply:
I agree. I will wrap it with non-async method like this.
```csharp
public static void ModifyAgentCrystal(Address agentAddress, BigInteger crystal) =>
    ModifyAgentCrystalAsync(agentAddress, crystal).Forget();
```